app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/VideoPlayerAdapter.java

```diff
@@ -33,13 +34,23 @@ public void pause() {
 
     @Override
     public void rewind() {
-        playbackController.rewind();
+        DisplayPreferences cachedPrefs = TvApp.getApplication() != null ? TvApp.getApplication().getCachedDisplayPrefs("usersettings") : null;
+
+        int skipMS = cachedPrefs != null && cachedPrefs.getCustomPrefs().get("skipBackwardLength") != null ?
+                Integer.parseInt(cachedPrefs.getCustomPrefs().get("skipBackwardLength")) : 10000;
+
+        playbackController.skip(-skipMS);
```

Comment:
This bit of code is now in 2 places; this file and the overlay fragment. Is it possible to put it in just one place without too much trouble?

Reply:
Yes, I could bump it back out to the playback controller I'm pretty sure.